tests/unit_tests/restruct_poc/CMakeLists.txt

```diff
@@ -9,6 +9,7 @@ target_sources(
     test_rotor.cpp
     test_math.cpp
     test_utilities.cpp
+    system/test_integrate_matrix.cpp
```

Comment:
Do we need to create a separate folder for this single test file?

Reply:
No. But my intent is for our unit test directory to have the same structure as our source directory so everything stays organized. I'll add more files in here for the tests of the other kernels, add a beams folder for those unit tests, etc. This is just step one so that we don't have to move the file later.